benchmark-scripts/Dockerfile.xpu

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2021-2022 Intel Corporation.
+# SPDX-License-Identifier: MIT
```

Comment:
should we use the same copyright and license header as the previous one?

Reply:
Yes, let me update to bsd 3